source-code/website/src/pages/marketplace/index.page.tsx

```diff
@@ -0,0 +1,272 @@
+import { Layout } from "../Layout.jsx"
+import { Meta, Title } from "@solidjs/meta"
+import { marketplaceItems } from "@inlang/marketplace"
+import { For, Show, Accessor, createSignal, createEffect } from "solid-js"
+import { Chip } from "#src/components/Chip.jsx"
+import { SearchIcon } from "../editor/@host/@owner/@repository/components/SearchInput.jsx"
+import { Button } from "../index/components/Button.jsx"
+import { GetHelp } from "#src/components/GetHelp.jsx"
+import Plus from "~icons/material-symbols/add-rounded"
+import Package from "~icons/material-symbols/package-2"
+
+type Category = "app" | "plugin" | "lintrule"
+
+const [searchValue, setSearchValue] = createSignal<string>("")
+
+const [selectedTags, setSelectedTags] = createSignal<Category[]>(["app", "plugin", "lintrule"])
+
+const filteredItems = () =>
+	marketplaceItems.filter((item: Record<string, any>) => {
+		console.log(item)
```

Comment:
@floriandwt seems like you have lints deactivated.

console.logs are linted. Remove the console log here please and ensure that you have lints activated in vscode

Reply:
weird, it got deactivated in my vscode, thanks for mentioning